app/tools/android/make_apk.py

```diff
@@ -442,7 +450,9 @@ def MakeApk(options, app_info, manifest):
     for dir_name in os.listdir(library_lib_path):
       lib_dir = os.path.join(library_lib_path, dir_name)
       if ContainsNativeLibrary(lib_dir):
+        compressed_lib = os.path.join(target_library_path, 'res', 'raw', NATIVE_LIBRARY + '.' + dir_name)
```

Comment:
I think this line exceed 80 chars, please follow http://google-styleguide.googlecode.com/svn/trunk/pyguide.html. 

Suggestion: You can enable pep8 style checking in you editor.